### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by tmjoen**
- plus a tiny bit of linting from upgrading linter configs
- dropped node 4 and 6 from travis builds

### CODE REVIEW

File: src/lib/transformer.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ export default functions => {
...
 
         return then(promises, () => {
-            node[prop] = values.toString();  
+            node[prop] = values.toString();
         });
     }
 
     function transform(node) {
-        if (node.type !== 'function' || !functions.hasOwnProperty(node.value))
+        if (node.type !== 'function' || !Object.prototype.hasOwnProperty.call(functions, node.value))
```
Comment (andyjansson) [inline src/lib/transformer.js:23]: Any specific reason for this change?

**Review by andyjansson — APPROVED** [review]
Looks good!

File: src/lib/transformer.js (hunk 1 above)
Comment (tmjoen) [reply src/lib/transformer.js:23]: https://eslint.org/docs/rules/no-prototype-builtins

Doesn't _really_ apply in this case, but it's a "best practice" thing and the linter complained, so I succumbed 😄

Comment (andyjansson) [reply src/lib/transformer.js:23]: Fair enough, thanks!

**Pull request merged**